tests/external-signed-ca-with-automatic-copy/install-server-with-external-ca-with-automatic-copy.yml

```diff
@@ -28,8 +27,8 @@
   hosts: ipaserver
   become: true
   vars:
-    ipaserver_external_cert_files_from_controller: "{{ groups.ipaserver[0] + '-chain.crt' }}"
-    #ipaserver_external_ca_file: "{{ groups.ipaserver[0] + '-cacert.asc' }}"
+    ipaserver_external_cert_files_from_controller: "{{ ansible_fqdn + '-chain.crt' }}"
```

Comment:
This is run on `localhost`. The use of `ansible_fqdn` is changing the names used for the certicate.

Reply:
Does this run on localhost? 3 lines above this change it says  `ipaserver`.

Reply:
O.k. you are right that here we have `hosts: ipaserver` here. But as you said that `groups.ipaserver[0]` in Azure will not be FQDN. That means that `groups.ipaserver[0]` used to create the certificate and `ansible_fqdn` here are not the same. It would be good to use the same variable here and also to generate the certificate.